backend/cloud_inquisitor/app.py

```diff
@@ -211,6 +213,19 @@ def __register_types(self):
                 logger.debug('Registered resource type {}'.format(cls.__name__))
         except SQLAlchemyError as ex:
             logger.warning('Failed loading type information: {}'.format(ex))
+
+    def __register_notifiers(self):
+        """Lists all notifiers to be able to provide metadata for the frontend
+
+        Returns:
+            `list` of `dict`
+        """
+        notifiers = {}
+        for ep in CINQ_PLUGINS['cloud_inquisitor.plugins.notifiers']['plugins']:
```

Comment:
What's ep?

Reply:
Entry Points

Reply:
Could we change the variable name please?  It's not immediately clear in this context like it is when you have `for ep in blah['foo']['entry_points']`.

Reply:
If we want to change that, we can do a refactor for clarity later, but this is an identical pattern we are using in quite a few places, so I'd prefer that to not be changed in one place, but rather be done as a refactor later

Reply:
I hear you, but as I mentioned above, the other cases explicitly call out `entry_points` in their `for in` construct.  This is not clear and can lead to confusion for others.